octobot_trading/data/trade.pxd

```diff
@@ -30,20 +30,19 @@ cdef class Trade:
     cdef public object side # TradeOrderSide
     cdef public object status # OrderStatus
     cdef public object trade_type # TraderOrderType
+    cdef public object taker_or_maker # ExchangeConstantsMarketPropertyColumns
```

Comment:
Is it not a str ?

Reply:
Yes it's a string, I used the order.pxd definition of it but it's also defined as an object. i will modify both.